### PR TITLE
Update env reference in registry builder

### PR DESCRIPTION
**Pull request by sbose78**

### CODE REVIEW

File: openshift-ci/Dockerfile.registry.build [hunk 1]
```diff
@@ -8,8 +8,9 @@ COPY manifests manifests
 COPY deploy/crds/*.yaml manifests/devconsole/${version}/
 
 USER root
-RUN sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/$(OPENSHIFT_BUILD_NAMESPACE)/stable:devconsole-operator," -i manifests/devconsole/${version}/devconsole-operator.v${version}.clusterserviceversion.yaml
+RUN ls /bin && sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator," -i manifests/devconsole/${version}/devconsole-operator.v${version}.clusterserviceversion.yaml
+RUN /bin/initializer
```
Comment (baijum) [inline openshift-ci/Dockerfile.registry.build]: Should we add `USER 1001` before this?

Comment (sbose78) [reply openshift-ci/Dockerfile.registry.build]: That's what I had, trying to see if that's what caused the issue.. hence moved it up..

Comment (baijum) [reply openshift-ci/Dockerfile.registry.build]: ok

**Comment by sbose78** [comment]
I'll see if I can get some answers on the CI log before we merge this :)

**Comment by baijum** [comment]
> I'll see if I can get some answers on the CI log before we merge this :)

ok!

**Comment by sbose78** [comment]
I think instead of
```
FROM quay.io/openshift/origin-operator-registry:latest
FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:latest
```
( which is horribly wrong anyway :D )
i should use 
```
FROM quay.io/openshift/origin-operator-registry:latest
```
So that openshift can override the first one and use the pipeline image. Probably, now the pipeline image isn't getting used. Rather `FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:latest` gets used which doesn't have the stuff inside `/bin`

**Comment by baijum** [comment]
> I think instead of
> 
> ```
> FROM quay.io/openshift/origin-operator-registry:latest
> FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:latest
> ```
> ( which is horribly wrong anyway :D )
> i should use
> 
> ```
> FROM quay.io/openshift/origin-operator-registry:latest
> ```
> So that openshift can override the first one and use the pipeline image. Probably, now the pipeline image isn't getting used. Rather `FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:latest` gets used which doesn't have the stuff inside `/bin`

That makes sense!

File: openshift-ci/Dockerfile.registry.build (hunk 1 above)
Comment (baijum) [reply openshift-ci/Dockerfile.registry.build]: Now, Do you want to move this down again?

**Comment by sbose78** [comment]
Taking this in, because the image build of operator-registry isn't in master of openshift/release.

**Pull request merged**